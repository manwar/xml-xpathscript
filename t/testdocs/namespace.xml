<doc>
    <foo:baz xmlns:foo="http://blah" />
    <bar:baz xmlns:bar="http://bleh" />
    <baz />
</doc>
