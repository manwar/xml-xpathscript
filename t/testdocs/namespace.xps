<%
    $template->namespace( 'http://blah' )->set( 'baz' => { pre => 'in foo' } );
    $template->namespace( 'http://bleh' )->set( 'baz' => { pre => 'in bar' } );
    $template->set( 'baz' => { pre => 'in nothing' } );
%>
<%= apply_templates() %>
