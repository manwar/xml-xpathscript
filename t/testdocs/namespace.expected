
<doc>
    in foo
    in bar
    in nothing
</doc>
